<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QWidget" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1658</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户操作界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidfet#centerwidget{background-color:66CCCC;};</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1661</width>
     <height>1031</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#e1d9cc;</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;用户管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>130</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 200, 205)</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;CCCC66</string>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>25</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 200, 205)</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 200, 205)</string>
    </property>
    <property name="text">
     <string>签收快递</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>1191</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
   </widget>
   <widget class="QPushButton" name="recharge">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 200, 205)</string>
    </property>
    <property name="text">
     <string>账号充值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeMessage">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 200, 205)</string>
    </property>
    <property name="text">
     <string>修改账号信息</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>331</width>
      <height>511</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
font: 12pt &quot;幼圆&quot;;</string>
    </property>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item>
     <property name="text">
      <string>根据快递时间查询</string>
     </property>
     <property name="toolTip">
      <string extracomment="gdgf"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>根据用户名查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>根据快递状态查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>根据快递单号查询</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sendExpress">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 200, 205)</string>
    </property>
    <property name="text">
     <string>发送快递</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 12pt &quot;隶书&quot;;</string>
     </property>
     <property name="text">
      <string>年：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="year">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="minimum">
      <number>1970</number>
     </property>
     <property name="maximum">
      <number>2030</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;隶书&quot;;</string>
     </property>
     <property name="text">
      <string>月：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="month">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;隶书&quot;;</string>
     </property>
     <property name="text">
      <string>日：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="day">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>请输入查询信息：</string>
    </property>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>username</zorder>
   <zorder>search</zorder>
   <zorder>sign</zorder>
   <zorder>lineEdit</zorder>
   <zorder>tableWidget</zorder>
   <zorder>recharge</zorder>
   <zorder>changeMessage</zorder>
   <zorder>treeWidget</zorder>
   <zorder>sendExpress</zorder>
   <zorder>label_6</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_returnbotton_clicked()</slot>
  <slot>on_findAllGoods_clicked()</slot>
  <slot>on_findUserMessage_clicked()</slot>
  <slot>on_choosebotton_doubleclicked()</slot>
  <slot>on_checkBalance_clicked()</slot>
  <slot>on_recharge_clicked()</slot>
  <slot>on_changeMessage_clicked()</slot>
  <slot>on_sendExpress_clicked()</slot>
 </slots>
</ui>
